examples/drupal_loadtest.rs

```diff
@@ -22,48 +22,48 @@
 //! See the License for the specific language governing permissions and
 //! limitations under the License.
 
+use goose::boilerplate;
+boilerplate!();
+
 use rand::Rng;
 use regex::Regex;
 
-use goose::GooseAttack;
-use goose::goose::{GooseTaskSet, GooseClient, GooseTask};
-
 fn main() {
     GooseAttack::initialize()
         .register_taskset(GooseTaskSet::new("AnonBrowsingUser")
             .set_weight(4)
-            .register_task(GooseTask::new(drupal_loadtest_front_page)
+            .register_task(GooseTask::new(task!(drupal_loadtest_front_page))
                 .set_weight(15)
                 .set_name("(Anon) front page")
             )
-            .register_task(GooseTask::new(drupal_loadtest_node_page)
+            .register_task(GooseTask::new(task!(drupal_loadtest_node_page))
```

Comment:
I think there should be a goose_ task! macro that also calls ::new. Then it’s way less redundant and also task! cannot be accidentally forgotten.

Reply:
I agree we can improve ergonomics like this. However, I'm not sure we need a `task!()` and a `goose_task!()` -- once we have the outer wrapper I'm not seeing a need for the inner wrapper -- so maybe it should just all be folded into `task!()`?

Let's explore once what we have is merged.

Reply:
What I meant was:

`.register_task(task!(drupal_loadtest_simple))`

Or

`.register_task(goose_task!(drupal_loadtest_simple))`